### PR TITLE
Bump spark-shell spark.executor.memory up to 1g (default is 512m).

### PR DESCRIPTION
**Pull request by andrewpalumbo**
also a couple of cosmetic fixes for driver version numbers.

Any reason not to move the  spark.executor.memory up ?

### CODE REVIEW

**Comment by dlyubimov** [comment]
I forget, did we fix MAHOUT_OPTS env  for spark shell in public version? Generally we should be able to run it with MAHOUT_OPT="-Dspark.executor.memory=whatever", right?

File: spark-shell/src/main/scala/org/apache/mahout/sparkbindings/shell/MahoutSparkILoop.scala [hunk 1]
```diff
@@ -45,6 +45,8 @@ class MahoutSparkILoop extends SparkILoop {
       conf.set("spark.executor.uri", execUri)
     }
 
+    conf.set("spark.executor.memory", "1g")
```
Comment (dlyubimov) [inline spark-shell/src/main/scala/org/apache/mahout/sparkbindings/shell/MahoutSparkILoop.scala]: Defaults should be overwritten from MAHOUT_OPTS, right? but it doesn't look like it checks if it is already set?

**Comment by andrewpalumbo** [comment]
No- we pushed that fix for 0.10.1

**Comment by dlyubimov** [comment]
ok

**Comment by andrewpalumbo** [comment]
thx for looking at it. gonna push now.

**Pull request closed without merging**